test/karma.conf.js

```diff
@@ -1,38 +1,27 @@
 module.exports = function(config) {
-  var
-    plugins = [],
-    detectBrowsers = {
-      enabled: false,
-      usePhantomJS: false
-    },
-    addBrowserLauncher = function(browser) {
-      plugins.push('karma-' + browser.toLowerCase() + '-launcher');
-    };
-
-  if (process.env.TRAVIS) {
-    // Travis needs to run them in Firefox.
-    // Additionally, the tests don't pass in Safari.
-    config.browsers = ['Firefox'];
-
-  } else {
-    // If no browsers are specified, we enable `karma-detect-browsers`
-    // This detects all browsers available for testing
-    plugins.push('karma-detect-browsers');
-    config.browsers = ['Chrome', 'Firefox', 'IE'];
-
-    detectBrowsers.enabled = true;
-    detectBrowsers.postDetection = function(browsers) {
+  var detectBrowsers = {
+    enabled: false,
+    usePhantomJS: false,
+    postDetection: function(browsers) {
       var i = browsers.indexOf('Safari');
+
       if (i !== -1) {
         browsers.splice(i, 1);
       }
 
       return browsers;
-    };
+    }
+  };
 
-    config.browsers.forEach(addBrowserLauncher);
-    plugins.push('karma-qunit');
-    config.plugins = plugins;
+  // On Travis CI, we can only run in Firefox.
+  if (process.env.TRAVIS) {
+    config.browsers = ['Firefox', 'travisChrome'];
+  }
+
+  // If no browsers are specified, we enable `karma-detect-browsers`
+  // this will detect all browsers that are available for testing
+  if (!config.browsers.length) {
+    detectBrowsers.enabled = true;
```

Comment:
By [default karma includes anything in plugins that matches the following: `karma-*`][1] . So the default works in cases like this where the plugin is named `karma-detect-browsers` or `karma-qunit`. So we don't have to explicitly define anything in the plugins array.

[1]: http://karma-runner.github.io/1.0/config/configuration-file.html